### ISSUE TITLE
HELP: Adventure type quest man.

### ISSUE THREAD

**Issue by ghost** [raGan262/Quester#66]
Hello, so I'm making an adventure/survival server and want to add quests (this plugin :D) But what I want to do is this:

Say there is a miner. You click on him and he says, like go and get 1 gold. You get gold and then go back to him, click on him and get a reward (say $500) and then if u click on him again he says something like, go and get 1 diamond. Then u go back on click on him and get a reward (say $1000)

And then it basically carries on, giving quests one after another, after completing the one before.
Etc.

Is this possible (and if so how?)

Thanks

**Comment by raGan262** [comment]
Yes it is. Are you familiar with basic functionality? For example creating quest for each task you want player to do and setting up objectives and rewards?
